token-service/src/test/java/org/p2p/token/service/TokenServiceApiTest.kt

```diff
@@ -17,41 +15,49 @@ import org.koin.core.context.startKoin
 import org.koin.dsl.module
 import org.koin.test.KoinTest
 import org.koin.test.inject
-import kotlin.math.sin
+import timber.log.Timber
 import kotlinx.coroutines.test.runTest
 import org.p2p.core.R
 import org.p2p.core.crashlytics.CrashLoggerModule
 import org.p2p.core.network.NetworkCoreModule
 import org.p2p.core.network.NetworkServicesUrlStorage
 import org.p2p.core.network.environment.NetworkEnvironment
 import org.p2p.core.network.environment.NetworkEnvironmentManager
-import org.p2p.market.price.api.request.MarketPriceItemRequest
-import org.p2p.market.price.api.request.MarketPriceRequest
-import org.p2p.market.price.api.response.NetworkChain
-import org.p2p.market.price.mock.NetworkEnvironmentStorageMock
-import org.p2p.market.price.repository.MarketPriceRepository
+import org.p2p.token.service.api.TokenServiceApiRepository
+import org.p2p.token.service.api.request.TokenServiceItemRequest
+import org.p2p.token.service.api.request.TokenServiceMetadataRequest
+import org.p2p.token.service.api.request.TokenServicePriceRequest
+import org.p2p.token.service.api.request.TokenServiceQueryRequest
+import org.p2p.token.service.api.response.TokenServiceNetworkResponse
+import org.p2p.token.service.mock.NetworkEnvironmentStorageMock
 
-@RunWith(AndroidJUnit4::class)
-class MarketPriceApiTest : KoinTest {
+@RunWith(JUnit4::class)
+class TokenServiceApiTest : KoinTest {
```

Comment:
test class should be named like the class that is tested 
`TokenServiceApiRepositoryTest`